### PR TITLE
[MNT-24195] Added fix for ags capability issue

### PR DESCRIPTION
**Pull request by SathishK-T**
https://hyland.atlassian.net/browse/MNT-24195

### CODE REVIEW

File: amps/ags/rm-community/rm-community-repo/source/java/org/alfresco/module/org_alfresco_module_rm/capability/policy/CreatePolicy.java [hunk 1]
```diff
@@ -42,10 +43,22 @@ public int evaluate(
...
 
         // get the destination node
         NodeRef destination = getTestNode(invocation, params, cad.getParameters().get(0), cad.isParent());
 
+        //get the recordType
+        for (Object qname : invocation.getArguments()) {
+            if (qname!= null && qname.equals(RecordsManagementModel.TYPE_RECORD_FOLDER))
+            {
+                recordType = (QName) qname;
+            }else if (qname!= null && qname.equals(RecordsManagementModel.TYPE_RECORD_CATEGORY))
```
Comment (suneet-gupta) [inline amps/ags/rm-community/rm-community-repo/source/java/org/alfresco/module/org_alfresco_module_rm/capability/policy/CreatePolicy.java:56]: Incorporate alfresco formatting

Comment (SathishK-T) [reply amps/ags/rm-community/rm-community-repo/source/java/org/alfresco/module/org_alfresco_module_rm/capability/policy/CreatePolicy.java]: Changes are done.

File: amps/ags/rm-community/rm-community-repo/source/java/org/alfresco/module/org_alfresco_module_rm/capability/policy/CreatePolicy.java [hunk 2]
```diff
@@ -42,10 +43,23 @@ public int evaluate(
...
+        //get the recordType
+        for (Object qname : invocation.getArguments()) {
+            if (qname!= null && qname.equals(RecordsManagementModel.TYPE_RECORD_FOLDER))
+            {
+                recordType = (QName) qname;
+            }
+            else if (qname!= null && qname.equals(RecordsManagementModel.TYPE_RECORD_CATEGORY))
+            {
+                recordType = (QName) qname;
+            }
```
Comment (suneet-gupta) [inline amps/ags/rm-community/rm-community-repo/source/java/org/alfresco/module/org_alfresco_module_rm/capability/policy/CreatePolicy.java:60]: Can we combine both if else conditions in a single if statement as both blocks seems to perform similar function

**Review by suneet-gupta — CHANGES_REQUESTED** [review]

File: amps/ags/rm-community/rm-community-repo/source/java/org/alfresco/module/org_alfresco_module_rm/capability/policy/CreatePolicy.java (hunk 2 above)
Comment (SathishK-T) [reply amps/ags/rm-community/rm-community-repo/source/java/org/alfresco/module/org_alfresco_module_rm/capability/policy/CreatePolicy.java:55]: yes, I have done the changes as per the comment.

**Review by suneet-gupta — APPROVED** [review]

**Pull request merged**